client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -38,6 +39,21 @@ class EditorConfirmationSidebar extends React.Component {
 		this.props.onPublish( true );
 	};
 
+	getBusyButtonLabel( publishButtonStatus ) {
+		switch ( publishButtonStatus ) {
+			case 'update':
+				return this.props.translate( 'Updating…' );
+			case 'schedule':
+				return this.props.translate( 'Scheduling…' );
+			case 'publish':
+				return this.props.translate( 'Publishing…' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Publishing' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor-confirmation/publishing-state)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).